./after_work.sh: line 15: ./test.py: No such file or directory
